src/docs/product/releases/health/index.mdx

```diff
@@ -28,27 +28,32 @@ Sessions are submitted to Sentry so you can track the usage and adoption of your
 
 ### Active Users
 
-Number of users that started the application at least once in the last 24 hours.
+Users that started the application at least once in the last 24 hours.
 
 ### Crash
 
-The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `handled:no`. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) and [Sampling](/platform-redirect/?next=/configuration/sampling/).
+The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `error.unhandled:true `. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) and [Sampling](/platform-redirect/?next=/configuration/sampling/).
```

Comment:
```suggestion
The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `error.unhandled:true `. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) or [Sampling](/platform-redirect/?next=/configuration/sampling/).
```